libsignal-service/src/account_manager.rs

```diff
@@ -163,11 +204,20 @@ impl<Service: PushService> AccountManager<Service> {
             pre_keys: pre_key_entities,
             signed_pre_key: signed_prekey_record.try_into()?,
             identity_key: *identity_key_pair.public_key(),
-            last_resort_key: if use_last_resort_key {
-                Some(PreKeyEntity {
-                    key_id: 0x7fffffff,
-                    public_key: "NDI=".into(),
-                })
+            pq_pre_keys: pq_pre_key_entities,
+            pq_last_resort_key: if use_last_resort_key {
+                log::warn!("Last resort Kyber key unimplemented");
```

Comment:
do we maybe want to address this in the same PR, or do you think it's fine like this?

Reply:
It should be fine for us for now. Let's make an issue for it, because it actually requires revamping the pre-key code quite a bit, and we might want to do that in one go.